### ISSUE TITLE
develop a document

### ISSUE THREAD

**Issue by VnTao** [maruohon/malilib#106]
Hello author, have you considered writing a development document

**Comment by maruohon** [comment]
Yes, there will most likely be developer documentation once the on-going mod rewrite is finished and those mod versions get released.
